charts/incubator/bookstack/values.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: ghcr.io/linuxserver/bookstack
+  pullPolicy: IfNotPresent
+  tag: v21.04.2-ls132
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the bookstack documentation
+# https://hub.docker.com/r/linuxserver/bookstack
+env: {}
+  # TZ:
+
+service:
+  port:
+    port: 80
+
+ingress:
+  enabled: false
+
+# persistence:
+#   config:
+#     enabled: false
+#     emptyDir:
+#       enabled: false
+#     mountPath: /config
```

Comment:
Why is this commented out?

Reply:
Because the [template](https://github.com/k8s-at-home/charts/blob/447b92702c3ca330a66e645991c72277e34bbb88/templates/chart/values.yaml) has it commented out. I'll uncomment it.